### DISCUSSION TITLE
Can I choose a different Schema

### DISCUSSION THREAD

**Discussion by jeyjeycodes** [supabase/supabase#3204] [Questions] [answered]
Hi All,

I have a different schema outside public. Is it possible to use this schema instead of just public and storage.
As you can see below I have an 'app' schema. I don't want to have to use public
<img width="259" alt="Screenshot 2021-09-14 at 21 09 48" src="https://user-images.githubusercontent.com/83839350/133326797-f510f5f6-c714-48e0-b7fa-79623d1d9520.png">
<img width="702" alt="Screenshot 2021-09-14 at 21 11 04" src="https://user-images.githubusercontent.com/83839350/133326948-dc1fd46d-caf3-4c95-a64a-b61c0143c492.png">

**Comment by prescience-data** [comment]
+1  🔼

Use case is that I want only `GRANT USAGE ON SCHEMA public TO authenticated;` on the `public` schema, but I want a second `webhooks` schema that I can grant `USAGE` on to `anon`.

**Comment by MildTomato** [comment, marked as answer]
Short answer is, yes. But you will not be able to see those tables or edit them in the Table Editor in the Supabase dashboard.

We are working on exposing non public schema on the Table Editor in the supabase dashboard.
it's been taking a while as we need to make sure all the UI still works and we've had to refactor quite a lot of stuff.

You can still interact with non public schema through the SQL editor in the dashboard, and the tables will also appear in the Database section of the dashboard.
